quarkus/tests/junit5/src/main/java/org/keycloak/Keycloak.java

```diff
@@ -190,6 +190,7 @@ public Keycloak(Path homeDir, String version, List<Dependency> dependencies, boo
     }
 
     private Keycloak start(List<String> args) {
+        args = List.of("start-dev", "--log-level=org.keycloak.quarkus.runtime.tracing:debug");
```

Comment:
@shawkins What's the intention here?

Reply:
Should not have been committed. Removed now.